packages/oui-modal/src/modal.html

```diff
@@ -4,24 +4,34 @@
       class="oui-icon oui-icon-close_thin oui-modal__close-button"
       type="button"
       aria-label="Exit"
-      ng-click="$ctrl.onDismiss()"></button>
+      ng-click="$ctrl.onDismiss()"
+      ng-disabled="$ctrl.loading">
+    </button>
   </div>
   <div class="oui-modal__body oui-modal__body_no-icon">
-    <h2 class="oui-modal__title" ng-bind="::$ctrl.title"></h2>
+    <div class="oui-modal__loader" ng-if="$ctrl.loading">
+      <oui-spinner></oui-spinner>
+    </div>
+    <h2 class="oui-modal__title"
+        ng-bind="::$ctrl.title"
+        ng-if="$ctrl.title">
```

Comment:
Since `$ctrl.title` is one-time binded, you can go with `ng-if="::!!$ctrl.title"`. 
AngularJS keep a watcher if the value is `undefined`.

Reply:
done